version.cake

```diff
@@ -15,8 +16,14 @@ using System.Text.RegularExpressions;
 using System.Xml.Linq;
 using System.Xml.XPath;
 
-var AndroidSdkRepoName = "microsoft/appcenter-sdk-android";
-var AppleSdkRepoName = "microsoft/appcenter-sdk-apple";
+const string AndroidSdkRepoName = "microsoft/appcenter-sdk-android";
+const string AppleSdkRepoName = "microsoft/appcenter-sdk-apple";
+
+var ReposToUpdate = new List<string>()
```

Comment:
This can be just local var. Or even no need to this array at all

Reply:
Fixed